docker/mms_docker_gpu/mxnet-model-server.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
```

Comment:
This script is duplicated across cpu and gpu. Can we have 1 script?

Reply:
Ya, I had another PR which addresses this. But since you bring it up here, I will just close this PR and merge these changes with the other PR. That way we track everything in one PR.